### DISCUSSION TITLE
would it be okay to sell a game that uses this addon?

### DISCUSSION THREAD

**Discussion by dRoycon** [ramokz/phantom-camera#392] [Question / Help Wanted] [answered]
I have never used addons before and am interested in using this, but don't understand how this stuff works when it comes to actually selling a game. So basically, is it okay to sell a game that uses this addon? Never used addons before so sorry if this question is silly

**Comment by ramokz** [comment, marked as answer]
Not a silly question at all.
The answer is: Of course!

[The license](https://github.com/ramokz/phantom-camera/blob/main/LICENSE) for the project is about as open as it gets, so in reality you may do whatever you want with the addon.

**Reply by dRoycon** [reply]
that's awesome thanks!
